modules/cnf-intel-n3000-optimize-data-plane.adoc

```diff
@@ -0,0 +1,39 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="cnf-understanding-intel-hardware-accelertor-card-for-openshift-container-platform_{context}"]
+= Understanding Intel hardware accelerator cards for OpenShift Container Platform
+
+Hardware accelerator cards from Intel accelerate 4G/LTE and 5G Virtualized Radio Access Networks (vRAN) workloads. This in turn increases in turn the overall compute capacity of a commercial off-the-shelf platform.
+
+The hardware accelerator cards detailed in this section are:
```

Comment:
Please remove this list because it is redundant.  If you believe customers need greater visibility, then consider changing the block titles to titles so that they appear in the TOC.